task_sdk/tests/conftest.py

```diff
@@ -23,4 +23,4 @@
 
 @pytest.hookimpl(tryfirst=True)
 def pytest_configure(config: pytest.Config) -> None:
-    config.inicfg["airflow_deprecations_ignore"] = ()
+    config.inicfg["airflow_deprecations_ignore"] = ()  # type: ignore[assignment,operator]
```

Comment:
```suggestion
    config.inicfg["airflow_deprecations_ignore"] = ""
```

Reply:
I fixed it in https://github.com/apache/airflow/pull/42985 with

```
config.inicfg["airflow_deprecations_ignore"] = []
```

Which is probably better.